src/main/features/certificateOfService/routes/file-upload.ts

```diff
@@ -43,35 +43,39 @@ export default express.Router()
           new DraftService().save(viewDraft, user.bearerToken).then(() => {
             res.redirect(Paths.documentUploadPage.uri)
           })
-        } else if (FileTypes.acceptedMimeTypes().indexOf(file.type) === -1) {
-          viewDraft.document.fileToUploadError = FileUploadErrors.WRONG_FILE_TYPE
-          new DraftService().save(viewDraft, user.bearerToken).then(() => {
-            res.redirect(Paths.documentUploadPage.uri)
-          })
         } else {
-          DocumentsClient.save(user.bearerToken, file).then((documentManagementURI) => {
-            const documentType: DocumentType = viewDraft.document.fileToUpload
-            let files: UploadedDocument[] = []
-            draft.document.uploadedDocuments.map((file) => files.push(new UploadedDocument().deserialize(file)))
+          FileTypes.isOfAcceptedMimeType(file.path).then(function (accepted) {
```

Comment:
It reads out kind of weird that we're checking if file path is of accepted mime type. What do you think about passing just the file as an input argument?

Reply:
Also please use an arrow function instead of an anonymous function. Arrow functions have more predictable scoping rules.

Reply:
Also since we're using `ErrorHandling` we can use an await instead of promise syntax, i.e.
```
const available = await FileTypes.isOfAcceptedMimeType(file.path)
```

Reply:
it is using the file.path to check the actual file. it is not just looking at the file path. I'll change to an arrow function and the error handling

Reply:
I know you are using the parameter correctly, I just meant that it reads out slightly awkwardly :) It's not that important, just a suggestion.